modules/csg/csg_shape.cpp

```diff
@@ -2235,6 +2255,7 @@ void CSGPolygon3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "path_interval", PROPERTY_HINT_RANGE, "0.01,1.0,0.01,exp,or_greater"), "set_path_interval", "get_path_interval");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "path_simplify_angle", PROPERTY_HINT_RANGE, "0.0,180.0,0.1"), "set_path_simplify_angle", "get_path_simplify_angle");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "path_rotation", PROPERTY_HINT_ENUM, "Polygon,Path,PathFollow"), "set_path_rotation", "get_path_rotation");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "path_rotation_uses_accurate_tangent"), "set_path_rotation_uses_accurate_tangent", "get_path_rotation_uses_accurate_tangent");
```

Comment:
This name is very verbose IMO, should be made shorter

Reply:
I've renamed it to `path_rotation_accurate`, hopefully that's not too big a step in the other direction